models/repo_permission.go

```diff
@@ -98,6 +98,11 @@ func (p *Permission) CanWriteIssuesOrPulls(isPull bool) bool {
 	return p.CanWrite(unit.TypeIssues)
 }
 
+// CanSeePrivateIssues returns true if the user is allowed to see private issues on the repo.
+func (p *Permission) CanSeePrivateIssues() bool {
+	return p.CanWrite(unit.TypeIssues)
```

Comment:
Again, I don't think that's the correct permission.
Correct me if I'm wrong.

Reply:
Ditto as my comment.  - Just a placebo, I agree that it should be more restrictive.

Reply:
What about repo admins + special team(e.g. security team) that is the only one that should be able to see all the private issues on a repo.

Reply:
I think only special right for team that allows to see private issues should be used, that would be set by default to owners team but could be set by admin also to any other team if needed

Reply:
https://github.com/go-gitea/gitea/pull/17711/commits/4706958a4ff11f45e1ba5e8753cdba67d35e41bc